images/base/scripts/common/setup.sh

```diff
@@ -83,7 +83,12 @@ fi
 
 # First install protoc
 cd /tmp
-git clone https://github.com/google/protobuf.git
+if [ x$MACHINE = xs390x ]
+then
+    git clone https://github.com/gongsu832/protobuf.git
```

Comment:
The original script used clone then checkout so I just followed that to make minimal changes. As for clone then checkout branch vs clone branch directly, it's two commands vs one command, doesn't make much difference to me either way.